posix/tcp/wh_server_tcp/wh_server_tcp.c

```diff
@@ -4,128 +4,260 @@
 
 #include <stdint.h>
 #include <stdio.h>  /* For printf */
-#include <string.h> /* For memset, memcpy */
+#include <stdlib.h> /* For atoi */
+#include <string.h> /* For memset, memcpy, strcmp */
 #include <unistd.h> /* For sleep */
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
 
 #include "wolfhsm/wh_error.h"
 #include "wolfhsm/wh_comm.h"
+#include "wolfhsm/wh_common.h"
 #include "wolfhsm/wh_message.h"
 #include "wolfhsm/wh_server.h"
+#include "wolfhsm/wh_server_keystore.h"
 #include "wolfhsm/wh_nvm.h"
 #include "wolfhsm/wh_nvm_flash.h"
 #include "wolfhsm/wh_flash_ramsim.h"
 #include "port/posix/posix_transport_tcp.h"
 
 /** Local declarations */
-static int wh_ServerTask(void* cf);
+static int wh_ServerTask(void* cf, const char* keyFilePath, int keyId,
+                         int clientId);
 
 enum {
-	ONE_MS = 1000,
+    ONE_MS         = 1000,
     FLASH_RAM_SIZE = 1024 * 1024,
 };
 
 #define WH_SERVER_TCP_IPSTRING "127.0.0.1"
 #define WH_SERVER_TCP_PORT 23456
-#define WH_SERVER_ID 56
+#define WH_SERVER_ID 57
 
-static int wh_ServerTask(void* cf)
+static int loadAndStoreKeys(whServerContext* server, whKeyId* outKeyId,
```

Comment:
```suggestion
static int _LoadAndStoreKeys(whServerContext* server, whKeyId* outKeyId,
```
Been trying to make all static functions start with _